deploy/k8s/thv.yaml

```diff
@@ -19,10 +19,18 @@ spec:
         app.kubernetes.io/name: toolhive
     spec:
       serviceAccountName: toolhive
+      securityContext:
```

Comment:
Fun fact, now we've released our super duper funky wunky operator, we should probably deprecate these old YAMLs. Or at least make a decision on whether we want to support both operator and non operator installs!